lib/galaxy/tool_util/linters/tests.py

```diff
@@ -38,11 +38,16 @@ def lint_tsts(tool_xml, lint_ctx):
             if len(assertions) == 0:
                 continue
             if len(assertions) > 1:
-                lint_ctx.error(f"Test {test_idx}: More than one {ta} found. Only the first is considered.")
+                lint_ctx.error(f"Test {test_idx}: More than one {ta} found. Only the first is considered.", node=test)
```

Comment:
This should be a warn, but unrelated change.

Reply:
```suggestion
                lint_ctx.warn(f"Test {test_idx}: More than one {ta} found. Only the first is considered.", node=test)
```

Reply:
Thanks for the suggestion @jmchilton .. should I apply it in a separate PR?

@gallardoalba .. its that you can have only one `assert_command` (`assert_stderr`, or `assert_stdout`) per test. Within these you can have an arbitrary number of assertions.

Reply:
Ahhh, I understand. Thanks a lot!